src/RoomsSDKAdapter.test.js

```diff
@@ -0,0 +1,59 @@
+import {isObservable} from 'rxjs';
+
+import RoomsSDKAdapter from './RoomsSDKAdapter';
+
+describe('Rooms SDK Adapter Interface', () => {
+  let roomsSDKAdapter;
+  let mockSDK;
+
+  beforeEach(() => {
+    mockSDK = {
```

Comment:
shouldn't this be globally defined? we are going to need this mockSDK in basically every unit test module.

Reply:
You are right, we will need to mock the sdk in every test suite. We could have a helper function that mocks the sdk, or use your global definition. I'm chose to only mock the part of the sdk that is used by the RoomsAdapter so the test setup is very specific for the tests that are run.